spec/models/contribution_spec.rb

```diff
@@ -95,71 +95,23 @@
     it 'returns true for a valid authorship hash' do
       expect(described_class.valid_fields?(authorship)).to be true
     end
-    it 'returns false for an authorship hash without "featured"' do
-      authorship['featured'] = nil
-      expect(described_class.valid_fields?(authorship)).to be false
-    end
-    it 'returns false for an authorship hash without "status"' do
-      authorship['status'] = nil
-      expect(described_class.valid_fields?(authorship)).to be false
-    end
-    it 'returns false for an authorship hash without "visibility"' do
-      authorship['visibility'] = nil
-      expect(described_class.valid_fields?(authorship)).to be false
-    end
```

Comment:
Why do these specs get removed?  It looks like they test the desired functionality, although the implementation details have changed.

Reply:
NM, may delete that comment ^^ when I see more details below.